modules/ROOT/nav.adoc

```diff
@@ -29,6 +29,8 @@
   *** xref:scopes.adoc[Scopes]
   *** xref:errors.adoc[Errors]
   *** xref:reconnection.adoc[Reconnection]
+  *** xref:java-version-support.adoc[Java Version Support]
```

Comment:
Just checking that you're including the hard-coded titles intentionally. If you don't need different titles in the TOC, leave these blank and they'll pick up the title from the file.